webhook/webhook.py

```diff
@@ -137,3 +142,68 @@ def handle_team_event(self, payload):
             logging.error("invalid payload received: {}".
                           format(str(payload)))
             return "invalid payload", 405
+
+    def handle_membership_event(self, payload):
+        """
+        Handle when a user is added, removed, or invited to an organization.

```

Comment:
Is this for users being added/removed/invited to an organization or a team? This docstring seems to imply the former, but the actual logic looks like it has to do with teams.

Reply:
It's for membership, I will change the docstring accordingly